task/git-clone/0.1/README.md

```diff
@@ -46,11 +42,12 @@ as well as
 
 ## Usage
 
-### `git-clone`
+If `revision` is not provided in the params in the taskrun
+then it will take the branch that is specified `default`
+in the respective git repository and does the required job.
```

Comment:
```suggestion
in the respective git repository/
```

Reply:
@chmouel Thanks for suggestions. 

> Perhaps update the sample too, to not use master

will update the `samples` wherever `git-clone` is used.